src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -688,7 +688,10 @@ ApiGlobe.prototype.pan = function pan(pVector) {
  * @return     {number}  The zoom level.
  */
 ApiGlobe.prototype.getZoomLevel = function getZoomLevel() {
-    return this.scene.getMap().getZoomLevel();
+    const theoricalZoom = this.scene.getMap().computeTheoricalZoom(this.scene.currentCamera(), this.getRange());
+    // const idNode = this.scene.gfxEngine.screenCoordsToNodeId();
+    // const realZoom = this.scene.getMap().getZoomLevelFromIdnode(idNode);
+    return theoricalZoom;
```

Comment:
Sorry Quoc, I edited my previous comment : the problem is only for the level 1, for the others levels, it finally seems to be fine. 
Maybe there is a little problem in the formula we use to calculate the zoom level ?